src/__init__.py

```diff
@@ -1,15 +0,0 @@
-# Copyright 2013 National Renewable Energy Laboratory (NREL)
-# 
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-# 
-#        http://www.apache.org/licenses/LICENSE-2.0
-# 
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-# 
-# ++==++==++==++==++==++==++==++==++==++==
```

Comment:
I have deleted the file not the copyright statement (that went away together). Do you meant that I am not allowed to delete a file from the project if I find out that it is actually not necessary or even worst it is disturbing?